src/ipa/rpi/vc4/data/imx708_wide.json

```diff
@@ -408,6 +412,9 @@
         {
             "rpi.sharpen": { }
         },
+        {
+            "rpi.focus": { }
+        },
```

Comment:
The libcamera folks are likely to prefer a separate commit to introduce ``rpi.focus`` into all the tuning files.